WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Customs Form/PrintCustomsFormsView.swift

```diff
@@ -0,0 +1,165 @@
+                    Text(Localization.printingMessage)
+                        .bodyStyle()
+                        .multilineTextAlignment(.center)
+                        .fixedSize(horizontal: false, vertical: true)
+                }
+                .frame(width: Constants.imageWidth)
+                .padding(.bottom, Constants.verticalSpacing)
+
+                if invoiceURLs.count == 1 {
+                    VStack(spacing: Constants.buttonSpacing) {
+                        Button(action: {
+                            guard let url = invoiceURLs.first else {
+                                return
+                            }
+                            showPrintingView(for: url)
+                        }, label: {
+                            Text(Localization.singlePrintButton)
+                        })
+                        .buttonStyle(PrimaryButtonStyle())
+
+                        saveForLaterButton
+                    }
+                    .padding(.horizontal, Constants.horizontalPadding)
+                } else {
+                    VStack(spacing: 0) {
+                        ForEach(Array(invoiceURLs.enumerated()), id: \.element) { index, url in
+                            HStack {
+                                Text(String(format: Localization.packageNumber, index + 1))
+                                Spacer()
+                                Button(action: {
+                                    showPrintingView(for: url)
+                                }, label: {
+                                    HStack {
+                                        Spacer()
+                                        Text(Localization.printButton)
+                                    }
+                                })
+                                .buttonStyle(LinkButtonStyle())
+                            }
+                            .padding(.leading, Constants.horizontalPadding)
+                            .frame(height: Constants.printRowHeight)
+                            Divider()
+                        }
+
+                        Spacer()
+
+                        saveForLaterButton
+                            .padding(.horizontal, Constants.horizontalPadding)
+                            .frame(maxWidth: .infinity)
+                    }
+                }
+
+                Spacer()
+            }
+        }
+        .navigationTitle(Localization.navigationTitle)
+        .fullScreenCover(isPresented: $showingInProgress) {
+            InProgressView(viewProperties: inProgressViewProperties)
+                .edgesIgnoringSafeArea(.all)
+        }
+    }
+
+    private var saveForLaterButton: some View {
+        Button(action: {
+            presentationMode.wrappedValue.dismiss()
+        }, label: {
+            Text(Localization.saveForLaterButton)
+        })
+        .buttonStyle(SecondaryButtonStyle())
+        .renderedIf(showsSaveForLater)
+    }
```

Comment:
Actually I extract this view into a separate component since it's used in 2 places and I don't want to repeat myself 😅. But yeah, I'm trying to keep the balance between keeping `body` flat while avoiding scattering too many small components since it can be hard to follow.